centos.org/jobs/katello-pipelines.yml

```diff
@@ -16,28 +16,29 @@
       - katello
 
 - project:
-    name: katello-release-4.2
+    name: katello-release-4.3
     jobs:
       - 'foreman-pipeline-{type}-{version}-{distro}-{action}'
     empty: ''
     version:
-      - '4.2'
+      - '4.3'
     distro:
       - centos7
       - centos8
```

Comment:
You should probably even drop this. The CentOS Linux 8 images no longer work so you'll run into that when you want to release the next 4.3.z. Same applies to 4.2 and 4.4.

```suggestion
```